src/StoreApi/Routes/V1/Products.php

```diff
@@ -281,6 +281,28 @@ public function get_collection_params() {
 			'validate_callback' => 'rest_validate_request_arg',
 		);
 
+		// If the $_REQUEST contains a taxonomy query, add it to the params and sanitize it.
+		foreach ( $_REQUEST as $param => $value ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Do you think we should check whether the slug after `tax_` is a real taxonomy on the site before adding this param?

Reply:
I don't see that as a problem, the parameter will be collected anyway but will be disregarded here 
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/5be9023bcbc1918f84e9951f0acca114dcca99d4/src/StoreApi/Utilities/ProductQuery.php#L101-L127

I don't have strong opinion on this though, just want to avoid two more operation (get all taxonomies, check against the parameter passed).